addon/components/selectable-terms-list.hbs

```diff
@@ -1,9 +1,9 @@
 <ul
   class="selectable-terms-list"
-  {{did-insert (perform this.load) @vocabulary @terms @termFilter}}
-  {{did-update (perform this.load) @vocabulary @terms @termFilter}}
+  {{did-insert (perform this.loadFilteredTerms)}}
```

Comment:
suggesting passing same input as `did-update` modifier 

` {{did-insert (perform this.loadFilteredTerms) this.topLevelTermsRelationshipPromise @termFilter @terms}}`